ompi/runtime/help-mpi-runtime.txt

```diff
@@ -119,3 +119,25 @@ Aborting...
 There was an error registering software performance counters (SPCs) as
 MPI_T performance variables.  Your job will continue, but SPCs will be
 disabled for MPI_T.
+#
+[spc: /dev/shm failed]
```

Comment:
Does this really need to be /dev/shm ?

Reply:
I suppose it doesn't have to be /dev/shm, but that's the default right now.  I think we fall back to the working directory if it's unavailable.